app/docs/md/learn/practices/testing.md

```diff
@@ -85,3 +85,40 @@ test('Shut down local server', async t => {
 })
 ```
 
+## Component Testing
```

Comment:
The capitalization of headings (at least in this page) appears to be using the sentence case standard:

```suggestion
## Component testing
```